JASP-Engine/JASP/R/descriptives.R

```diff
@@ -158,6 +158,28 @@ Descriptives <- function(jaspResults, dataset, options) {
       }
     }
   }
+
+  # Pie charts
+  if (options$descriptivesPiechart) {
+
+    if(is.null(jaspResults[["pieCharts"]])) {
+      jaspResults[["pieCharts"]] <- createJaspContainer("Pie charts")
```

Comment:
Perhaps a little bit more concise:
```r
if(is.null(jaspResults[["pieCharts"]])) {
      jaspResults[["pieCharts"]] <- createJaspContainer("Pie charts", dependencies = c("splitby", "descriptivesPiechart"), position = 9)
```

Reply:
ok, but the original version is used trough the whole script. Should I rewrite the rest of descriptives.R to be like this suggestion?

Reply:
Nah that's overkill. There isn't really a preferred way of doing this (yet).